www/docs/support-us/index.php

```diff
@@ -11,5 +11,74 @@
 
 use MySociety\TheyWorkForYou\Renderer\Markdown;
 
+# define array of payment amounts and current/one-off options
+# note - new annual and monthly payments need to be defined as new prices in stripe
+# e.g. 'donate_monthly_10'.
+$payment_amounts = [
+    'monthly' => [
+        '2' => '£2',
+        '5' => '£5',
+        '10' => '£10',
+    ],
+    'annually' => [
+        '5' => '£5',
+        '10' => '£10',
+        '50' => '£50',
+    ],
+    'one-off' => [
+        '5' => '£5',
+        '10' => '£10',
+        '50' => '$50',
+    ],
+];
+
+
+$default_amounts = [
+    'monthly' => '5',
+    'annually' => '10',
+    'one-off' => '10',
+];
+
+
+$default_type = 'annually';
+
+# use the how-often parameter if set, if not default to option at end of line (options are 'monthly', 'annually', or 'one-off')
+$payment_type = get_http_var('how-often', $default_type);
```

Comment:
Please add a check here that it is one of the valid options, otherwise we can get errors when bots spider with random crap in the parameters.